src/puppet-web/firer.ts

```diff
@@ -207,7 +207,7 @@ async function checkRoomJoin(m: Message): Promise<void> {
         const loaded = inviteeContactList[i] instanceof Contact
 
         if (!loaded) {
-          let c = room.member(inviteeList[i])
+          let c = room.member({remark: inviteeList[i]}) || room.member({nick: inviteeList[i]})
```

Comment:
> We can keep room.member(inviteeList[i]) here, after you follow the review for member() parameters suggestion.

I don't agree with this suggestion, because of the following reason:

We keep function `room.member()` work, but it should work only with the function find member by nickName.    
We cannot use it both find remark and nickName, because one can set a remark same as nick, then it would be confused. 

So, what do you think about it?